pkg/database/migrations.go

```diff
@@ -1900,3 +1911,13 @@ func (db *Database) MigrateTo(ctx context.Context, target string, rollback bool)
 	}
 	return m.Migrate()
 }
+
+// multiExec is a helper that executes the given sql clauses against the tx.
+func multiExec(tx *gorm.DB, sqls ...string) error {
```

Comment:
@whaught unrelated, but I got annoyed writing migrations...

Reply:
nice - we do this a lot